locals.tf

```diff
@@ -0,0 +1,4 @@
+locals {
+  name = "ingress-nginx"
+  prove_port = "10254"
```

Comment:
@heldersepu is this is a typo ? Did you mean "probe" port ? The variable name is consistent everywhere, the code works just fine. But maybe it is "probe" ?

Reply:
Yep that is a typo